### DISCUSSION TITLE
Deletion Requests for the Autopilot object for use as an complete Offboarding tool

### DISCUSSION THREAD

**Discussion by tobiasssandberg** [okieselbach/Autopilot-Manager#24] [Ideas]
Hi @okieselbach and thanks for an amazing tool!

Could it be possible to add another function, like the **Deletion Requests**, but also for the **Autopilot object**. If that would be possible the Autopilot Manger could also be used as an Offboarding tool (like the https://github.com/ugurkocde/IntuneOffboarding) and we could leverage the solution as both Onboarding and Offboarding. I think that would be amazing.

Let me know if there is anything I can do to help make that possible.

**Comment by okieselbach** [comment]
Would be possible, but currently I do not have plans to extend it in that way.
